tools/devops/automation/templates/build/api-diff-process-results.yml

```diff
@@ -73,7 +73,12 @@ steps:
     } catch {
       $converted = $inputContents + "`n`nUnable to convert markdown: $_`n`n"
     }
-    $githubComments = New-GitHubCommentsObject -Org "xamarin" -Repo "xamarin-macios" -Token $Env:GITHUB_TOKEN -Hash $Env:GIT_HASH
+
+    $uri = $(Build.Repository.Uri)
+    $orgRepo = $uri.Replace("https://github.com/", "")
+    $org, $repo = $orgRepo -split "/",2
+
+    $githubComments = New-GitHubCommentsObject -Org $org -Repo $repo -Token $Env:GITHUB_TOKEN -Hash $Env:GIT_HASH
```

Comment:
Can't you just make `New-GitHubCommentsObject` take a url instead and then do the splitting in that function? That would remove all the duplicated code (or create a new function - `New-GitHubCommentsObjectWithUrl`)

Reply:
Make sense, I'll do that